build.sbt

```diff
@@ -213,15 +213,41 @@ lazy val `quill-cassandra` =
     )
     .dependsOn(`quill-core-jvm` % "compile->compile;test->test")
 
+lazy val `quill-cassandra-datastax` = 
+   (project in file("quill-cassandra-datastax"))
+    .settings(commonSettings: _*)
+    .settings(mimaSettings: _*)
+    .settings(
+      fork in Test := true
+    )
+    .dependsOn(`quill-cassandra` % "compile->compile;test->test")
+
 lazy val `quill-cassandra-monix` =
   (project in file("quill-cassandra-monix"))
+    .settings(commonSettings: _*)
+    .settings(mimaSettings: _*)
+    .settings(
+      fork in Test := true
+    )
+    .dependsOn(`quill-cassandra-datastax` % "compile->compile;test->test")
+    .dependsOn(`quill-monix` % "compile->compile;test->test")
+
+lazy val `quill-cassandra-lagom` =
+   (project in file("quill-cassandra-lagom"))
     .settings(commonSettings: _*)
     .settings(mimaSettings: _*)
     .settings(
       fork in Test := true,
+      libraryDependencies ++= {
+        val lagomVersion = "1.5.0-RC1"
+        Seq(
+          "com.lightbend.lagom" %% "lagom-scaladsl-persistence-cassandra" % lagomVersion % Provided,
```

Comment:
why `Provided`?

Reply:
Lagom's version keeps changing. I thought it might be better for use to determine their main version.